pkg/controller/servicebindingrequest/servicebinder.go

```diff
@@ -279,7 +346,6 @@ func (b *serviceBinder) bind() (reconcile.Result, error) {
 	if _, err = b.updateServiceBindingRequest(sbr); err != nil {
 		return noRequeue(err)
```

Comment:
The sbr should be requeued on any update error, otherwise the removal of the finalizers above won't be available in the API server. The side effect of this is the garbage collector won't be able to remove the resource.

```go
return requeueError(err)
```

Reply:
handled it here 0526003